### PR TITLE
docs: 📜 SCM Compliance changes

### PR DESCRIPTION
**Pull request by kjellhaaland**
### Short summary

Updated readme
Renamed CONTRIBUTING file

Link to issue: https://github.com/equinor/lighthouse/issues/550

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,60 @@
...
+## Contributing ⚒️
 
-```PS
-$ pnpm install
-$ pnpm start
-```
+Contributing, check out our [contributing guide](./CONTRIBUTING.md)
 
-# Development
+Also, check out the [lighthouse team docs](https://github.com/equinor/lighthouse-docs) for information on how to stucture work.
```
Comment (Gustav-Eikaas) [inline README.md:33]: typo in stucture, correct to structure

File: README.md [hunk 2]
```diff
@@ -1,74 +1,60 @@
...
-    - [Optional Software](#optional-software)
-- [How to run](#how-to-run)
-- [Development](#development)
-- [Create new Release](#create-new-release)
-    - [Deployment dev environment](#deploy-to-dev)
-    - [Deployment production environment](#deploy-to-production)
+This repository contains various applications and many different Power BI reports. Most of the applications are read-only, except *Release Control*, *Scope Change Request* and partially *Piping and Heat trace* (also called Discipline Release Control). The backend for *Release Control*, *Scope Change Request* and *Piping and Heat Trace* can be found [here](https://github.com/equinor/lighthouse-scope-change-control-api).
 
-## About
+This project uses [NodeJS](https://nodejs.org/en/) and [PNPM](https://pnpm.io/). Applications are developed using [React](https://react.dev/), [Typescript](https://www.typescriptlang.org/) and [Vite](https://vitejs.dev/).
```
Comment (Gustav-Eikaas) [inline README.md:12]: Can we just add a quick mention that its hosted in Radix

**Review by Gustav-Eikaas — APPROVED** [review]
Looks good too me

File: README.md (hunk 2 above)
Comment (kjellhaaland) [reply README.md:12]: Added link to Radix :)

File: README.md (hunk 1 above)
Comment (kjellhaaland) [reply README.md:33]: Fixed

**Pull request merged**